### ISSUE TITLE
Finding the instructions a little hard to follow

### ISSUE THREAD

**Issue by SteveALee** [bradygaster/azure-tools-vscode#11]
I'm finding the  [extension page instructions](https://marketplace.visualstudio.com/items?itemName=bradygaster.azuretoolsforvscode) difficult to follow. I'm not being picky but am not so familiar with everything MS. I am sure others are the same.

* I'm not using AAD and have no idea at this point what a tenant is or if I use any of @outlook.com, @live.com, or @hotmail.com. I just login to the portal when prompted

* I assume MSA = Microsoft Account? = @outlook.com, @live.com, or @hotmail.com?

* The instructions are gif loops which I personally find harder to follow than text. It's not clear where the start of the loop is and they are a bit fast. For accessibility you should provide [a text alternative](http://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html) for the gifs.

* It is of course possible to figure out that we use 'Azure login' in code and that the extension then makes the process really clear. However it should be much easier to get going

* What is the life time of the login?

* I do not have the globe region icon in my statusbar. No update is being offered for my extension. I'm in the UK so would like to set the region

Let me know if should pull some of these out into new issues

**Comment by bradygaster** [comment]
This is extremely good feedback we'll take into account and use as fodder to update our README. Or, we'll add a doc series to the repository so folks can review documentation as we progress. 

If you update to the 0.0.9 version you'll see the globe (i think i addressed your inability to update in your other issue). 

We aren't persisting the token anywhere [yet]. Each code instance requires a new login. We're trying to keep the token-persistence functionality to a minimum while in preview. If enough people like the extension and make use of it we'll definitely need to address token-management and cross-instance persistence. I'll leave this issue open until we make a better instructional experience for people to use to get logged in. 

Thanks so much for your feedback, @SteveALee!

**Comment by SteveALee** [comment]
> If you update to the 0.0.9 version you'll see the globe 

Yes turns out I was on 0.0.5 so manually uninstalled and updated (no upgrade offered in the extension list). I can confirm the globe now available.

> We're trying to keep the token-persistence functionality to a minimum while in preview

Sounds good. I guess we could use the cloud icon in the status bar to easily tell if we are logged in or not. My concern was logging in twice really. Persisting the region as well would  be good.

re giffs. Perhaps a static pause at the start or better, don't auto run them. (have a giff overlay like FaceBook does)

**Comment by bradygaster** [comment]
@SteveALee we updated the README and made the instructions a lot more lower-level, with links to some of the deeper videos inline rather than a series of large animated gifs. Please let me know if this makes things easier to follow. Feel free to send a pull request if you feel this could be improved upon. We're open to suggestions. For now, closing this issue

**Comment by SteveALee** [comment]
@bradygaster That's really very good and much easier to follow.

Just 2 minor points which I'll look into providing a PR for
* How you get the azure.tenantId value is still not totally clear and doesn't quite read well for me.
* Server farms are a new concept for me in Azure Functions.

Thanks for such a great response

**Comment by SteveALee** [comment]
Sorry, looks like I never responded. I did, however, look at the
improvements. That's really good thanks!!!
